extensions/campeonato-brasileiro/package.json

```
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "campeonato-brasileiro",
+  "title": "Campeonato brasileiro",
+  "description": "View the latest Campeonato Brasileiro standings.",
+  "icon": "icon.png",
+  "author": "danielsequeira",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "standings",
+      "title": "Standings",
+      "description": "View the latest Campeonato Brasileiro standings.",
+      "mode": "view"
+    },
+    {
+      "name": "topScorers",
+      "title": "View Top scorers",
```

```suggestion
      "title": "View Top Scorers",
```
Same here with the Title convention 🙂